partiql-lang/src/main/kotlin/org/partiql/lang/syntax/PartiQLParserBuilder.kt

```diff
@@ -29,12 +30,25 @@ import org.partiql.lang.types.CustomType
  */
 class PartiQLParserBuilder {
```

Comment:
Should we plan on moving this to the parser package or should we just go directly to deprecate and remove pattern here?

Reply:
It exists in the new package. This one will be deprecated then removed.